climada/hazard/centroids/test/test_centr.py

```diff
@@ -96,7 +98,46 @@ def test_to_crs(self):
 class TestCentroidsReader(unittest.TestCase):
     """Test read functions Centroids"""
 
-    def test_centroid_pass(self):
+    def test_from_csv(self):
+        """Read a centroid csv file correctly"""
+        tmpfile = Path('test_write_csv.csv')
+        test_crs = CRS.from_user_input(4326).to_wkt()
```

Comment:
@sarah-hlsn : for the default, please use the `DEF_CRS` variable. Also, add one test for a non-default crs.